graviti/portex/field.py

```diff
@@ -5,14 +5,16 @@
 """Portex record field releated classes."""
 
 
-from typing import Any, Dict, Iterable, List, Mapping, Tuple, Union
+from typing import Any, Dict, Iterable, List, Mapping, Tuple, TypeVar, Union
 
 import pyarrow as pa
 
 from graviti.portex.base import _INDENT, PortexType
 from graviti.portex.package import Imports
 from graviti.utility import NameOrderedDict
 
+_T = TypeVar("_T", bound="Fields")
```

Comment:
Where should the `TypeVar` be? Inside or outside of the class? Any differences?

Reply:
If the TypeVar is inside the class, it cannot be used in the class method.

Reply:
And inside the class will give the TypeVar a more explicit scope.